doc/functions.xml

```diff
@@ -294,6 +294,22 @@ merge:"diff3"
 
 </section>
 
+<section xml:id="sec-debug">
+  <title>Debugging Nix Expressions</title>
+
+  <para>Nix is a unityped, dynamic language, this means every value can
```

Comment:
unityped or untyped?

Reply:
Neither of those are true :grin:

Reply:
@shlevy Hm, why is nix not unityped?

Reply:
`[]` and `{}` are different types. The fact that those are checked at eval time (which, FWIW, can be seen as compile time with respect to the underlying `.drv` outputs...) doesn't make them not typed!

Reply:
Ah, well, I see eval time more as execution time.